lmdeploy/cli/serve.py

```diff
@@ -255,7 +242,6 @@ def gradio(args):
     def api_server(args):
```

Comment:
when add `--model-name` to `lmdeploy serve api_server` with workspace turbomind model path, the server model_id does not change
```
lmdeploy serve api_server internvl2-2b-tb/ --model-name bigmodel 
2024-07-26 20:28:14,455 - lmdeploy - WARNING - internvl2-2b-tb/ seems to be a turbomind workspace, which can only be ran with turbomind engine.
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
[WARNING] gemm_config.in is not found; using default GEMM algo
HINT:    Please open http://0.0.0.0:23333 in a browser for detailed api usage!!!
HINT:    Please open http://0.0.0.0:23333 in a browser for detailed api usage!!!
HINT:    Please open http://0.0.0.0:23333 in a browser for detailed api usage!!!
```

```
curl http://0.0.0.0:23333/v1/models | grep id
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   566  100   566    0     0   184k      0 --:--:-- --:--:-- --:--:--  276k
{"object":"list","data":[{"id":"/mnt/141/InternVL2-2B/snapshots/3f4a31bf77bef03ee9e783e709f678ee13b2bfc4","object":"model","created":1721998167,"owned_by":"lmdeploy","root":"/mnt/141/InternVL2-2B/snapshots/3f4a31bf77bef03ee9e783e709f678ee13b2bfc4","parent":null,"permission":[{"id":"modelperm-FDBBQkrsKamxmiqcVRJjqJ","object":"model_permission","created":1721998167,"allow_create_engine":false,"allow_sampling":true,"allow_logprobs":true,"allow_search_indices":true,"allow_view":true,"allow_fine_tuning":false,"organization":"*","group":null,"is_blocking":false}]}]}
```
``

Reply:
Fixed